pulp_rpm/app/modelresource.py

```diff
@@ -0,0 +1,135 @@
+from pulpcore.app.modelresource import QueryModelResource
+from pulp_rpm.app.models import (
+    DistributionTree,
+    Modulemd,
+    ModulemdDefaults,
+    Package,
+    PackageCategory,
+    PackageGroup,
+    PackageEnvironment,
+    PackageLangpacks,
+    RepoMetadataFile,
+    UpdateRecord,
+)
+
+
+# class <model-name>Resource(QueryModelResource):
+#
+#     def __init__(self, repo_version):
+#         super().__init__(repo_version)
+#         self.queryset = <model>.objects.filter(pk__in=self.repo_version.content)
+#
+#     class Meta:
+#         model = <model>
+class DistributionTreeResource(QueryModelResource):
+
+    def __init__(self, repo_version):
+        super().__init__(repo_version)
+        self.queryset = DistributionTree.objects.filter(pk__in=self.repo_version.content)
+
+    class Meta:
+        model = DistributionTree
+
+
+class ModulemdResource(QueryModelResource):
+
+    def __init__(self, repo_version):
+        super().__init__(repo_version)
+        self.queryset = Modulemd.objects.filter(pk__in=self.repo_version.content)
+
+    class Meta:
+        model = Modulemd
+
+
+class ModulemdDefaultsResource(QueryModelResource):
+
+    def __init__(self, repo_version):
+        super().__init__(repo_version)
+        self.queryset = ModulemdDefaults.objects.filter(pk__in=self.repo_version.content)
```

Comment:
It might be a little more efficient if you use:

```pk__in=self.repo_version.content.filter(pulp_type=ModulemdDefaults.get_pulp_type()```

Reply:
Actually, do we have an index on ```pulp_type```? If not, we definitely should...

https://github.com/pulp/pulpcore/blob/master/pulpcore/app/models/base.py#L93